notebooks/Data_visualization/Data_visualizations.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>Can you add a cell above mentioning that there are different graphs to select from based on the data type we are trying to visualize and in this notebook we explore some of the most popular/common ones. It would also be worthwhile to mention a brief description (one liner perhaps) on what each type of graph is before diving into the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>